granulate_utils/linux/cgroups/exceptions.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+from pathlib import Path
+
+
+class SkippedCgroup(Exception):
+    def __init__(self, current_cgroup: str, new_cgroup_name: str) -> None:
+        super().__init__(f"Skipping creating new {new_cgroup_name} cgroup since current cgroup is: {current_cgroup}")
```

Comment:
```suggestion
        super().__init__(f"Skipping the creation of new {new_cgroup_name!r} cgroup since the current cgroup is {current_cgroup!r}")
```